std_msgs/Header header

uint16 control_mode

uint32 id

float32 acc_x
float32 acc_y
float32 acc_z

float32 bodyrate_x
float32 bodyrate_y
float32 bodyrate_z


float32 position_x
float32 position_y
float32 position_z

float32 quad_w
float32 quad_x
float32 quad_y
float32 quad_z


